Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -631,7 +631,6 @@ void FieldProbe::ComputeDiags (int step)
             amrex::Vector<int> displs_vector;
             if (amrex::ParallelDescriptor::IOProcessor()) {
                 displs_vector.resize(mpisize, 0);
```

Comment:
This is a bug, resize only sets _new_ elements to the value provided.

https://en.cppreference.com/w/cpp/container/vector/resize
```suggestion
                displs_vector.assign(mpisize, 0);
```
https://en.cppreference.com/w/cpp/container/vector/assign